pkg/authorization/api/synthetic.go

```diff
@@ -10,4 +10,6 @@ const (
 	NodeMetricsResource = "nodes/metrics"
 	NodeStatsResource   = "nodes/stats"
 	NodeLogResource     = "nodes/log"
+
+	RestrictedEndpointsResource = "endpoints/restricted"
```

Comment:
@deads2k any better ideas for the virtual resource to do authz checks against to cover restricted endpoint addresses?